AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participants-list/participants-list.component.html

```diff
@@ -34,16 +34,23 @@ <h3 class="govuk-heading-m">{{ isEditMode ? 'Participants' : 'Participants added
     <div *ngFor="let pat of participants">
       <div *ngIf="!pat.is_judge">
         <div class="govuk-grid-row vh-row">
-          <div class="govuk-grid-column-two-thirds vhtable-header">
+          <div *ngIf="isInterpreter(pat) || isInterpretee(pat); else noImage" class="govuk-grid-column-one-third vh-image-10">
+            <img src="/assets/images/interpreter.png" height="30" width="30" class="govuk-header__logotype-crest" alt="Interpreter link" />
```

Comment:
You can add the width and the height to your stylesheet (above)

```
.vh-image-10 {
    width: 15%;
    padding-left: 10%;
    img {
        height: 30px;
        width: 30px;
    }
}
```

Reply:
updated as above.